pkg/core/blockchain.go

```diff
@@ -1050,6 +1050,23 @@ func (bc *Blockchain) processNEP5Transfer(cache *dao.Cached, tx *transaction.Tra
 	}
 }
 
+// GetTransferLog returns UTXO transfer log for the acc.
+func (bc *Blockchain) GetTransferLog(acc util.Uint160) *state.TransferLog {
```

Comment:
It can be huge, there is some paging needed.

Reply:
Fixed. Made similar change for nep5 transfer log.